ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/DLModel.java

```diff
@@ -342,4 +343,10 @@ public ModelTensors parseModelTensorOutput(Output output, ModelResultFilter resu
         return tensorOutput;
     }
 
+    public Settings getClusterSettings() {
```

Comment:
https://github.com/opensearch-project/ml-commons/blob/6d33e917fa4e04b40fb58b9972300452354bc418/common/src/main/java/org/opensearch/ml/common/settings/MLFeatureEnabledSetting.java#L34


We usually add all our settings here. Can you please check how other settings are propagated and follow accordingly?

Reply:
I have a question. Would you like the Batch Processing to be enabled or disabled with a boolean feature from `MLFeatureEnabledSetting`? 
The Batch Size, however, is passed from the [MLCommonsSettings.java](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/settings/MLCommonsSettings.java)